bravado/client.py

```diff
@@ -280,7 +280,7 @@ def construct_request(operation, request_options, **op_kwargs):
     """
     url = operation.swagger_spec.api_url.rstrip('/') + operation.path_name
     request = {
-        'method': operation.http_method.upper(),
+        'method': str(operation.http_method.upper()),
```

Comment:
encode('ascii') converts a unicode string to bytes in python3, and this throws ConnectionError (possibly due to encoding assumed by the petstore server?). Added a test case with requests pinned to < 2.8, and it fails for py35 and py36 with the above code changes.